Makefile

```diff
@@ -302,7 +302,7 @@ updateSemver: # unless the package.json has been manually edited.
 dist/commonjs: bin/traceur.js
 	./traceur --dir src/ dist/commonjs/ --modules=commonjs
 
-prepublish: bin/traceur.js bin/traceur-runtime.js
+prepublish: bin/traceur.js bin/traceur-runtime.js dist/commonjs/
 
```

Comment:
I guess this is a convenience for you, it's not something we use in the make or npm builds right?